### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elijahbenizzy**
Smoothed out the first two sentences

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 # Hamilton
 
-The general purpose micro-orchestration framework for creating [dataflows](https://en.wikipedia.org/wiki/Dataflow) from python functions! That is, your single tool to express things like data, ML, LLM pipelines/workflows, and even web request logic!
+The general purpose micro-orchestration framework for building [dataflows](https://en.wikipedia.org/wiki/Dataflow) from python functions. Express data, ML, LLM pipelines/workflows, and web requests with a clear, easy-to-follow API.
```
Comment (skrawcz) [inline README.md:23]: functions > API.

Comment (elijahbenizzy) [reply README.md:23]: yes, we mention functions in the prior sentence

Comment (skrawcz) [reply README.md:23]: API is a confusing term for me here.

Comment (elijahbenizzy) [reply README.md:23]: Calling it "interface"

**Review by skrawcz — APPROVED** [review]

**Pull request merged**